packages/terra-overlay/src/Overlay.jsx

```diff
@@ -76,14 +82,16 @@ class Overlay extends React.Component {
   setContainer(node) {
     if (!node) { return; } // Ref callbacks happen on mount and unmount, element is null on unmount
     this.overflow = document.documentElement.style.overflow;
+    const selector = this.props.rootSelector;
 
     if (this.props.isRelativeToContainer) {
       this.container = node.parentNode;
       this.containerChildren = this.container.children;
       this.disableContainerChildrenFocus();
+      this.containerChildren[0].focus();
     } else {
       document.documentElement.style.overflow = 'hidden';
-      this.container = null;
+      document.querySelector(selector).setAttribute('aria-hidden', 'true'); // prevent screen reader from moving to hidden content
```

Comment:
Does this need to be removed when the overlay is hidden or unmounted.

Reply:
I do see that iit's remove on the resteBeforeOverlay method, but I'm curious if we should add it to the componentWillUnmount lifecycle step. Anyone else have any thoughts?

Reply:
I don't think it would hurt to check in componentWillUnmount lifecycle step and make sure its removed.

Reply:
I talked to @bjankord, and most of these items are related to fixing a bug with the fullscreen overlay that was actually fixed at the same time in [#1955](https://github.com/cerner/terra-core/pull/1955), and this stuff was in addition to the main scope of this PR, so I'm going to strip it all out in favor of the already merged changes. :)

Reply:
For checking this attribute, the merged changes from master run this check again in componentWillUnmount, so this seems to be addressed in master.